vllm/model_executor/model_loader/loader.py

```diff
@@ -12,8 +12,10 @@
 import huggingface_hub
 import numpy as np
 import torch
+from gguf import MODEL_ARCH_NAMES, get_tensor_name_map
```

Comment:
Could you switch this to `import gguf` to help with clarity and use the imports more explicitly like `gguf.MODEL_ARCH_NAMES` and `gguf.get_tensor_name_map`?